kibana/values.yaml

```diff
@@ -44,8 +44,17 @@ kibanaConfig: {}
 #       nestedkey: value
 
 # If Pod Security Policy in use it may be required to specify security context as well as service account
-podSecurityContext: {}
-  #runAsUser: "place the user id here"
+
+podSecurityContext:
+  fsGroup: 2000
```

Comment:
What is group id 2000? It doesn't exist in the kibana docker image. The kibana group has id 1000.

Reply:
Thought I'd use a number that does not map to an actual group (still works) but yes let's go with 1000 that maps to an existing group instead!